website/content/docs/security/acl/acl-system.mdx

```diff
@@ -323,13 +323,16 @@ rules:
 | [`agent`](/docs/acl/acl-rules#agent-rules)          | Utility operations in the [Agent API](/api/agent), other than service and check registration                                                                                                             |
 | [`event`](/docs/acl/acl-rules#event-rules)          | Listing and firing events in the [Event API](/api/event)                                                                                                                                                 |
 | [`key`](/docs/acl/acl-rules#key-value-rules)        | Key/value store operations in the [KV Store API](/api/kv)                                                                                                                                                |
-| [`keyring`](/docs/acl/acl-rules#keyring-rules)      | Keyring operations in the [Keyring API](/api/operator/keyring)                                                                                                                                           |
+| [`keyring`](/docs/acl/acl-rules#keyring-rules)      | Keyring operations in the [Keyring API](/api/operator/keyring)
+| [`mesh`](/docs/acl/acl-rules#mesh-rules)            | Access to ingress gateways, terminating gateways and mesh configuration entries [Config API](/api/config)
```

Comment:
What do we have that uses `mesh` rules? I feel like I haven't seen that in the documentation. For example, if I go to the config HTTP API page, it doesn't show any `mesh` rules at all... are these just not documented yet?

(And `mesh` is obviously missing from this experiment so far because it's not in the source spreadsheet at all)